Mathlib/Topology/Order/LeftRightNhds.lean

```diff
@@ -56,9 +56,8 @@ theorem TFAE_mem_nhdsWithin_Ioi {a b : α} (hab : a < b) (s : Set α) :
   · intro h
     rcases mem_nhdsWithin_iff_exists_mem_nhds_inter.1 h with ⟨v, va, hv⟩
     rcases exists_Ico_subset_of_mem_nhds' va hab with ⟨u, au, hu⟩
-    refine ⟨u, au, fun x hx => ?_⟩
-    refine' hv ⟨hu ⟨le_of_lt hx.1, hx.2⟩, _⟩
-    exact hx.1
+    refine ⟨u, au, fun x hx ↦ ?_⟩
+    refine hv ⟨hu ⟨le_of_lt hx.1, hx.2⟩, hx.1⟩
```

Comment:
```suggestion
    exact ⟨u, au, fun x hx ↦ hv ⟨hu ⟨le_of_lt hx.1, hx.2⟩, hx.1⟩
```

Reply:
@adomani sooner or later we will take care of all these terminal `refine`, right?